scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -1033,15 +1033,15 @@ class ScalaTestAntTask extends Task {
   }
   
   private class NestedSuiteElement {
-    private var suiteId: String = null
+    private var suiteId: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.